internal/pkg/encoding/json/schema/schema_test.go

```diff
@@ -80,21 +80,123 @@ func TestValidateObjects_Ignore_TestConnectionButton(t *testing.T) {
 	invalidSchema := []byte(`
 {
   "type": "object",
-  "$schema": "http://json-schema.org/draft-04/schema#",
+  "required": ["id"],
   "properties": {
+    "id": {"type": "object"},
     "test_connection": {
-      "type": "button"
+      "type": "button",
+      "format": "sync-action",
+      "propertyOrder": 30,
+      "options": {
+        "async": {
+          "label": "TEST CONNECTION",
+          "action": "validate_connection"
+        }
+      }
     }
```

Comment:
V prvom commite som nakopiroval do testov nove custom definicie, ktore mi na Slack-u nalinkoval JKotek.

Reply:
Dolezite vylepsenie je, ze som pridal do definicie `"id": {"type": "object"},`, tj. kontrolujem aj to, ze ignorovanim custom definitions sa nerozbije ostatna validacia.